clustered-cache/src/main/java/org/jboss/as/quickstarts/datagrid/eap/subsystem/web/JsfController.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2018, Red Hat, Inc. and/or its affiliates, and individual
+ * contributors by the @authors tag. See the copyright.txt in the
+ * distribution for a full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.as.quickstarts.datagrid.eap.subsystem.web;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.Resource;
+import javax.enterprise.inject.Model;
+import javax.enterprise.inject.Produces;
+import javax.inject.Named;
+
+import org.infinispan.Cache;
+import org.infinispan.container.entries.CacheEntry;
+import org.jboss.logging.Logger;
+
+/**
+ * A simple JSF controller to show how the RHDG cache configured with EAP infinispan subsystem is used.
+ *
+ * @author <a href="mailto:wfink@redhat.com">Wolf-Dieter Fink</a>
+ */
+@Model
+public class JsfController {
+  private static final Logger LOGGER = Logger.getLogger(JsfController.class);
+  private CacheView cacheView;
+
+//   @Resource(lookup = "java:jboss/infinispan/container/jdg-container")
+//   CacheContainer container;   // container could be injected if needed
+
+  /**
+   * The direct lookup like
+   *
+   * @Resource(lookup = "java:jboss/infinispan/cache/jdg-container/EAPcache")
+   *
+   * is not working because of https://issues.redhat.com/browse/WFLY-11817.
```

Comment:
I would instead link to the corresponding JBEAP jira:
https://issues.redhat.com/browse/JBEAP-22290

Reply:
done